.github/workflows/docker_image_build.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
 
   release:
     name: Build and Push to Registry
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
this change has the potential to break the workflow in future since it's a dangling label. what about fixing it to 24.x instead?

Reply:
IMHO:

In any case we should watch out for changes in GitHub runners, but:
- pinning the version we have to do a PR in any case
- not pinning the version we shall do a PR only if something is breaking (which I see ok), and restoring the x-1 version

In short, I suggest to spend time on these details only if we need to. On the other hand, we should keep an eye on this coming changes from providers

Reply:
I like `ubuntu-latest` because it forces us to fix/upgrade our workflows as soon as possible!